src/http/common.cr

```diff
@@ -246,7 +246,7 @@ module HTTP
   # timezone `GMT` (interpreted as `UTC`).
   #
   # ```
-  # HTTP.format_time(Time.new(2016, 2, 15)) # => "Sun, 14 Feb 2016 21:00:00 GMT"
+  # HTTP.format_time(Time.new(2016, 2, 15)) # => "Mon, 15 Feb 2016 00:00:00 GMT"
```

Comment:
The output depends on local time zone configuration. `Time.new` should better receive a specific time zone.